components/Canvas.js

```diff
@@ -3,67 +3,120 @@ import styled from "styled-components";
 
 function lineWidthRandomizer() {
   const randomNumber = Math.random();
-  if (randomNumber <= 0.4) {
+  if (randomNumber <= 0.7) {
     return 2;
-  } else if (randomNumber <= 0.8) {
+  } else if (randomNumber <= 0.9) {
     return 8;
   } else {
     return 4;
   }
 }
 
-let firstLineWidth = lineWidthRandomizer();
-let secondLineWidth = lineWidthRandomizer();
-let thirdLineWidth = lineWidthRandomizer();
-let fourthLineWidth = lineWidthRandomizer();
-
-export default function CanvasOne() {
+export default function Canvas() {
   const canvasRef = useRef(null);
-  const [backgroundColor, setBackgroundColor] = useState(148);
+
+  const [color, setColor] = useState(148);
+  const [rotation, setRotation] = useState(45);
+  const [outline, setOutline] = useState(0);
+  const [variation, setVariation] = useState([
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+  ]);
+
   useEffect(() => {
     const canvas = canvasRef.current;
     const context = canvas.getContext("2d");
+
     // Beginn drawing statements
-    context.fillStyle = `hsl(${backgroundColor}, 60%, 95%)`;
+    context.fillStyle = `hsl(${color}, 60%, 95%)`;
     context.fillRect(0, 0, canvas.width, canvas.height);
 
-    context.fillStyle = "hsl(148, 100%, 50%)";
+    context.fillStyle = `hsl(${color}, 100%, 50%)`;
     context.translate(150, 150);
-    context.rotate((0 * Math.PI) / 180);
+    context.rotate((rotation * Math.PI) / 180);
     context.fillRect(-40, -40, 80, 80);
 
-    context.lineWidth = firstLineWidth;
-
-    context.strokeStyle = `hsl(${backgroundColor}, 60%, 95%)`;
+    context.lineWidth = variation[0];
+    context.strokeStyle = `hsl(${color}, 60%, 95%)`;
     context.strokeRect(-22.5, -22.5, 45, 45);
 
-    context.strokeStyle = "hsl(148, 100%, 50%)";
-    context.lineWidth = secondLineWidth;
-    context.strokeRect(-60, -60, 120, 120);
+    context.strokeStyle = `hsl(${color}, 100%, 50%)`;
+
+    for (let i = 0; i <= outline - 1; i++) {
+      let x = -60 - 20 * i;
+      let y = -60 - 20 * i;
+      let width = 120 + 40 * i;
+      let height = 120 + 40 * i;
+      context.lineWidth = variation[i];
+      context.strokeRect(x, y, width, height);
+    }
 
-    context.lineWidth = thirdLineWidth;
-    context.strokeRect(-80, -80, 160, 160);
+    // context.lineWidth = variation.secondLineWidth;
+    // context.strokeRect(-60, -60, 120, 120);
 
-    context.lineWidth = fourthLineWidth;
-    context.strokeRect(-100, -100, 200, 200);
+    // context.lineWidth = variation.thirdLineWidth;
+    // context.strokeRect(-80, -80, 160, 160);
+
+    // context.lineWidth = variation.fourthLineWidth;
+    // context.strokeRect(-100, -100, 200, 200);
 
     // Reset the transformation matrix to the identity matrix
```

Comment:
please delete the comments before committing your code:)

Reply:
I have forgotten to delete it ... sorry :)